test/helpers/fixture_deploy_helper.rb

```diff
@@ -65,7 +67,7 @@ def deploy_dirs_without_profiling(dirs, wait: true, allow_protected_ns: false, p
     protected_namespaces: nil, render_erb: false, allow_globals: true)
     kubectl_instance ||= build_kubectl
 
-    deploy = KubernetesDeploy::DeployTask.new(
+    deploy = Krane::DeprecatedDeployTask.new(
```

Comment:
This is the basis of the integration tests, so it should use one of the real public interfaces. There may be other changes associated with using `Krane::` (and having the globals command available might help), so that could be a separate PR, but I'd do that very soon.

Reply:
Just changed to address it, I agree we should still use `KubernetesDeploy` API while Krane 1.0 is not released.